composer.json

```diff
@@ -11,24 +11,16 @@
         }
     ],
     "homepage": "https://github.com/ITK-Leantime/leantime-tickettemplate",
-    "require": {
-        "ergebnis/composer-normalize": "^2.43"
-    },
     "require-dev": {
+        "ergebnis/composer-normalize": "^2.43",
         "friendsofphp/php-cs-fixer": "^3.51",
-        "leantime/leantime": "^3.2",
+        "leantime/leantime": "^3.4.5",
```

Comment:
Should `leantime/leantime` not be a normal requirement?

Reply:
No because it is solely used for code analysis, if it isn't installed code analysis complains about a lot of missing functions.